src/NuGet.Clients/NuGet.VisualStudio.Implementation/NuGet.VisualStudio.Implementation.csproj

```diff
@@ -9,10 +9,10 @@
     <Description>Implementation of the NuGet.VisualStudio extensibility APIs.</Description>
     <Guid>9623cf30-192c-4864-b419-29649169ae30</Guid>
     <ImportedFromTypeLib>NuGet.VisualStudio.Implementation</ImportedFromTypeLib>
+    <LangVersion>8</LangVersion>
```

Comment:
This PR uses C#8 NRT. Is there a scenario where a compatible compiler would not be available?

Reply:
I think this is ok.. @zivkan do you know ?